app/views/profiles/_form.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= simple_form_for(@profile) do |f| %>
```

Comment:
pls wrap this in a container and add the red button to f.submit

Reply:
added button in partial